src/content/docs/setup/configuration/multistore-setup.mdx

```diff
@@ -0,0 +1,295 @@
+
+## Implementation Steps
+
+<Tasks>
+
+<Task>
+### Verify Environment Setup
+
+Before implementing multi-store functionality, confirm your environment is properly configured.
+
+<Steps>
+1. **Check the `fstab.yaml` file** in your repository to ensure it's linked to the Document Authoring environment at da.live, as described in <Link href="https://experienceleague.adobe.com/developer/commerce/storefront/merchants/get-started/multistore/#folder-mapping" text="multistore/#folder-mapping" />
+
+2. **Verify the `config.json` file** is connected to your Adobe Commerce instance to enable content preview, publishing, and editing as described in <Link href="https://experienceleague.adobe.com/developer/commerce/storefront/merchants/get-started/multistore/#site-config" text="multistore/#site-config" />
+
+</Steps>
+
+<Aside type="caution" title="Environment setup">
+This verification step is crucial as it ensures you can properly preview and publish content for your new store views.
+</Aside>
+
+</Task>
+
+<Task>
+### Create Content Folder Structure
+
+Set up the folder structure in your Document Authoring environment to support the new store views.
+
+<Steps>
+1. **Navigate to the connected content folder** in da.live.
+
+2. **Create the Canadian English folder**:
+   - Select "New folder"
+   - Name it `en-ca`
+
+3. **Create the French folder**:
+   - Select "New folder"  
+   - Name it `fr-ca`
+
+4. **Copy content from the main root folder** into each new store view folder:
+   - Select all content from the main root folder
+   - Copy the content
+   - Navigate to the `en-ca` folder and paste
+   - Repeat for the `fr-ca` folder
+</Steps>
+
+<Aside type="note" title="Copying content">
+Exclude the newly created store view folders when copying content to avoid nested duplicates.
+</Aside>
+
+</Task>
+
+<Task>
+### Configure Adobe Commerce Store Views
+
+Create the corresponding store views in Adobe Commerce Admin to match your content folder structure.
+
+<Steps>
+1. **Access Adobe Commerce Admin** and navigate to Stores > Settings > All Stores.
+
+2. **Create the Canadian English store view**:
+   - Follow the store view creation process
+   - Configure as: one website, one store, one store view
+   - Set the store view code to match your folder name (`en-ca`)
+
+3. **Create the Canadian French store view**:
+   - Repeat the process for the French locale
+   - Set the store view code to match your folder name (`fr-ca`)
+</Steps>
+
+<Aside type="note" title="Store view codes">
+The store view codes in Adobe Commerce Admin must exactly match the folder names created in da.live for proper integration.
+</Aside>
+
+</Task>
+
+<Task>
+### Update Storefront Configuration
+
+Modify your project's configuration files to support the new store views.
+
+<Steps>
+1. **Update the `fstab.yaml` file** to include new folder mappings:
+
+    ```yaml
+    mountpoints:
+    /: https://example.mountpoint.com/:f:/r/sites/myproject
```

Comment:
Is this supposed to be indented or is this how the file is actually formatted? Same for lines 126 and 127.

For example, see https://commerce-docs.github.io/microsite-commerce-storefront/merchants/get-started/multistore/#folder-mapping.